openmetadata-ui/src/main/resources/ui/src/utils/QueryBuilderElasticsearchFormat.js

```diff
@@ -0,0 +1,390 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/* eslint-disable */
+
+import { defaultConjunction } from 'react-awesome-query-builder/lib/utils/defaultUtils';
```

Comment:
Can we rename this file to: `openmetadata-ui/src/main/resources/ui/src/utils/QueryBuilderElasticsearchFormatUtils.ts`
To make it consistent with others